HelloWorld/test.java

```diff
@@ -0,0 +1,45 @@
+package com.ej.alumni.admin.zendesk.model;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+@JsonInclude(Include.ALWAYS)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ZendeskTicketVia {
+  private static final String CHANNEL = "channel";
+  private static final String SOURCE = "source";
+  private static final String TO = "to";
+  private static final String FROM = "from";
+  private static final String ADDRESS = "address";
+  private static final String NAME = "name";
+  private String channel = "";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename field "channel" to prevent any misunderstanding/clash with field "CHANNEL" defined on line 11. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.alumni-services-001.com/coding_rules#rule_key=squid%3AS1845)